pkcs12.go

```diff
@@ -73,9 +73,13 @@ type UnsupportedFormat string
 func (e UnsupportedFormat) Error() string { return string(e) }
 
 // ConvertToPEM converts all "safe bags" contained in pfxData to PEM blocks.
-func ConvertToPEM(pfxData []byte, password string) (blocks []*pem.Block, err error) {
-	p, err := bmpString(password)
-	password = ""
+func ConvertToPEM(pfxData []byte, utf8Password []byte) (blocks []*pem.Block, err error) {
+	p, err := bmpString(utf8Password)
+
+	for i := 0; i < len(utf8Password); i++ {
+		utf8Password[i] = 0
+	}
+
 	if err != nil {
 		return
```

Comment:
The error from bmpString will say "string contains characters...".

Should this code amend the error to indicate that it's the password causing an error?